spec/models/canonical/sync/ingredients_spec.rb

```diff
@@ -101,7 +101,7 @@
         it 'adds alchemical properties' do
           perform
           expect(item_in_json.alchemical_properties.pluck(:name))
-            .to eq ['Weakness to Frost', 'Fortify Sneak', 'Weakness to Poison', 'Fortify Restoration']
+            .to contain_exactly('Weakness to Frost', 'Fortify Sneak', 'Weakness to Poison', 'Fortify Restoration')
```

Comment:
It turns out these are not deterministically ordered in order of priority. However, this test is not testing the priority order, it is only testing that the associations are created. It's better for that purpose to test that the array contains the right elements, rather than that the order is the same.